package.json

```diff
@@ -13,7 +13,8 @@
     "copy": "cp CNAME storybook-static/",
     "build-storybook": "build-storybook",
     "chromatic": "npx chromatic --project-token ugo6c655mvm --auto-accept-changes",
-    "link:reaflow": "yarn unlink && yarn link && cd node_modules/react && yarn unlink && yarn link && cd ../react-dom && yarn unlink && yarn link"
+    "link:reaflow": "yarn link && cd ./node_modules/react && yarn link && cd ../react-dom && yarn link",
```

Comment:
What is this for?

Reply:
It's for collaborators of Reaflow only, it's to quickly link a custom project to the reaflow project.

This way, one can work on his own repo (e.g: a demo) using a local reaflow and iterate much more quickly.

Reply:
I think given this is pretty specific to your workflow ( i handle this differently ) , I'd rather not include this in the core at this time.

Reply:
How do you do this then? Because using npm/yarn linking is very common for npm projects.

Reply:
Let's remove this from core plz

Reply:
I did but I still don't think it's the right choice. IMHO it'll make contributing harder, while it doesn't really cause any trouble for anyone not trying to contribute.